osu.Game.Rulesets.Osu/Mods/OsuModZoomed.cs

```diff
@@ -0,0 +1,92 @@
+// Copyright (c) ppy Pty Ltd <contact@ppy.sh>. Licensed under the MIT Licence.
+// See the LICENCE file in the repository root for full licence text.
+
+using System;
+using osu.Framework.Bindables;
+using osu.Framework.Graphics.Sprites;
+using osu.Framework.Localisation;
+using osu.Framework.Utils;
+using osu.Game.Configuration;
+using osu.Game.Graphics.UserInterface;
+using osu.Game.Overlays.Settings;
+using osu.Game.Rulesets.Mods;
+using osu.Game.Rulesets.Scoring;
+using osu.Game.Rulesets.UI;
+using osu.Game.Scoring;
+using osuTK;
+
+namespace osu.Game.Rulesets.Osu.Mods
+{
+    internal class OsuModZoomed : Mod, IUpdatableByPlayfield, IApplicableToScoreProcessor
+    {
+        public override string Name => "Zoomed";
+        public override string Acronym => "ZM";
+        public override IconUsage? Icon => FontAwesome.Solid.Glasses;
+        public override ModType Type => ModType.Fun;
+        public override LocalisableString Description => "Big brother is watching your cursor.";
+        public override double ScoreMultiplier => 1;
+        public override Type[] IncompatibleMods => new[] { typeof(OsuModBarrelRoll) };
```

Comment:
playing with the Zoomed Mod with FL enabled currently hard crashes the game , might want to make them temporarily incompatible

Reply:
Fixed.